client/src/partials/depots_management/depots_management.html

```diff
@@ -0,0 +1,121 @@
+<!-- breadcrumb header -->
+<bh-breadcrumb
+  path="DepotCtrl.bcPaths"
+  button="DepotCtrl.bcButtons">
+</bh-breadcrumb>
+
+<!-- main content -->
+<div class="flex-content">
+  <div class="container-fluid">
+    <div class="row">
+
+      <div class="col-md-6">
+        <div class="panel panel-default">
+          <div class="panel-heading">
+            {{ "DEPOT.DEPOTS_LIST" | translate }}
+          </div>
+          <table class="table table-bordered table-condensed">
+            <thead>
+              <tr>
+                <th>{{ "DEPOT.REFERENCE" | translate }}</th>
+                <th>{{ "DEPOT.NAME" | translate }}</th>
+                <th>{{ "DEPOT.IS_WAREHOUSE" | translate }}</th>
+                <th></th>
```

Comment:
Do you mean to have an extra `<th>`?  If so, the proper way to do this is to use `<th colspan="2">` on the "IS_WAREHOUSE" column.